docs/guides/billing.md

```diff
@@ -82,4 +82,46 @@ If you're gating access to the entire app, you should check for billing:
 
 **Note**: the merchant may refuse payment when prompted or cancel subscriptions later on, but the app will already be installed at that point. We recommend using [billing webhooks](https://shopify.dev/docs/apps/billing#webhooks-for-billing) to revoke access for merchants when they cancel / decline payment.
 
+## Canceling a subscription
+
+With the `cancel` method you'll be able to cancel a single subscription. First, you'll need to obtain the id for the subscription you wish to cancel, using the `subscriptions` method.
```

Comment:
I guess not yet, but we should update this to say "you can also grab the id from the request method".

Reply:
Working on that separate PR ... will update as part of that one.